docs/openapi/components/schemas/common/TradeLineItem.yml

```
@@ -62,6 +62,7 @@ properties:
   countryOfOrigin:
     title: Country of Origin
     description: Identify the 2-digit ISO country code of the country of production.
```

There are no digits (numerals) here...
```suggestion
    description: Identify the 2-character ISO country code of the country of production.
```